tests/buf/oom.c

```diff
@@ -40,6 +41,9 @@ void test_buf_oom__grow(void)
 	cl_assert(git_buf_oom(&buf));
 
 	git_buf_dispose(&buf);
+#else
+    cl_skip();
+#endif
```

Comment:
You know, we should just use our custom allocators here. Create a custom test allocater that only has $n$ bytes available, plug it in and then run this test. This should result in a "real" OOM without us having to trick the host. I also think it would be quite helpful in other tests to verify exceptional behaviour.

Obviously, that doesn't have to be part of this PR.

Reply:
Ooooooh, that's a good idea.

Reply:
Addressing this in a separate pullrequest - forthcoming.